src/javascript/app_2/Modules/settings/components/menu/menu_item.jsx

```diff
@@ -27,7 +26,7 @@ const MenuItem = ({ title, description, img_src, path }) => {
 
 MenuItem.propTypes = {
     description: PropTypes.string,
-    img_src    : PropTypes.string,
+    Icon       : PropTypes.func,
```

Comment:
@easteregg 
It'd be better to use `ProptTypes.element` since it should be a react element.